src/NServiceBus.Core.Tests/Recoverability/MoveToErrorsExecutorTests.cs

```diff
@@ -18,23 +18,25 @@ public void Setup()
         {
             dispatcher = new FakeDispatcher();
             staticFaultMetadata = new Dictionary<string, string>();
-            moveToErrorsExecutor = new MoveToErrorsExecutor(dispatcher, ErrorQueueAddress, staticFaultMetadata);
+            moveToErrorsExecutor = new MoveToErrorsExecutor(dispatcher, staticFaultMetadata, headers => { });
         }
 
         [Test]
-        public async Task MoveToErrorQueue_should_dispatch_message_to_error_queue()
+        [TestCase(ErrorQueueAddress)]
+        [TestCase("random_error_queue")]
```

Comment:
do we need  testcases here? At first glance `"random_error_queue"` should be good enough to verify the usage of the passed parameter?